[2025-04-02T21:48:21.300+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Proj.Create_Cassandra_schema manual__2025-04-02T21:47:46.930657+00:00 [queued]>
[2025-04-02T21:48:21.308+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Proj.Create_Cassandra_schema manual__2025-04-02T21:47:46.930657+00:00 [queued]>
[2025-04-02T21:48:21.309+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-02T21:48:21.322+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Create_Cassandra_schema> on 2025-04-02 21:47:46.930657+00:00
[2025-04-02T21:48:21.327+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2025-04-02T21:48:21.327+0000] {standard_task_runner.py:60} INFO - Started process 1896 to run task
[2025-04-02T21:48:21.332+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Weather_Proj', 'Create_Cassandra_schema', 'manual__2025-04-02T21:47:46.930657+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks.py', '--cfg-path', '/tmp/tmp1i_vxnsq']
[2025-04-02T21:48:21.333+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask Create_Cassandra_schema
[2025-04-02T21:48:21.376+0000] {task_command.py:423} INFO - Running <TaskInstance: Weather_Proj.Create_Cassandra_schema manual__2025-04-02T21:47:46.930657+00:00 [running]> on host f47524c1cda8
[2025-04-02T21:48:21.442+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Weather_Proj' AIRFLOW_CTX_TASK_ID='Create_Cassandra_schema' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T21:47:46.930657+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-02T21:47:46.930657+00:00'
[2025-04-02T21:48:21.443+0000] {cassandra_creator.py:4} WARNING - Cluster.__init__ called with contact_points specified, but no load_balancing_policy. In the next major version, this will raise an error; please specify a load-balancing policy. (contact_points = ['cassandra'], lbp = None)
[2025-04-02T21:48:21.448+0000] {cassandra_creator.py:5} WARNING - Downgrading core protocol version from 66 to 65 for 172.24.0.4:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Cluster.protocol_version
[2025-04-02T21:48:21.450+0000] {cassandra_creator.py:5} WARNING - Downgrading core protocol version from 65 to 5 for 172.24.0.4:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Cluster.protocol_version
[2025-04-02T21:48:21.461+0000] {policies.py:289} INFO - Using datacenter 'datacenter1' for DCAwareRoundRobinPolicy (via host '172.24.0.4:9042'); if incorrect, please specify a local_dc to the constructor, or limit contact points to local cluster nodes
[2025-04-02T21:48:21.503+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/WeatherScripts/cassandra_creator.py", line 12, in create_schema
    session.execute("""
  File "cassandra/cluster.py", line 2677, in cassandra.cluster.Session.execute
  File "cassandra/cluster.py", line 4956, in cassandra.cluster.ResponseFuture.result
cassandra.protocol.SyntaxException: <Error from server: code=2000 [Syntax error in CQL query] message="line 14:25 no viable alternative at input '/' (... double,            temp_max_celisus double,            humidity_gram[/]...)">
[2025-04-02T21:48:21.513+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=Weather_Proj, task_id=Create_Cassandra_schema, execution_date=20250402T214746, start_date=20250402T214821, end_date=20250402T214821
[2025-04-02T21:48:21.523+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 7 for task Create_Cassandra_schema (<Error from server: code=2000 [Syntax error in CQL query] message="line 14:25 no viable alternative at input '/' (... double,            temp_max_celisus double,            humidity_gram[/]...)">; 1896)
[2025-04-02T21:48:21.542+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-04-02T21:48:21.566+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
